MAINTAINING.md

```diff
@@ -0,0 +1,147 @@
+# RIOT Review Guidelines
+
+The items presented in the following list should be checked for each PR review,
+in this order. This list is however not exhaustive and defines a rather minimal
+baseline on things that should be checked.
+
+Note: Any of the items in this list might be skipped, if **good** reasons are presented.
+
+Also note: This reviewing order is meant to maximize efficiency and minimize overhead, since failing in step n makes steps n+x obsolete. Maintainers should be aware that starting a review with 4. (Code Style) not only wasts their own time, but also the time of the author
+
+## 1. - Review the fundaments
+
+1.1 Does the reasoning for this PR make sense?
+1.2 Do the concepts used by the PR make sense? 
+1.3 Does the PR break with existing concepts?
+1.4 Is the structure of the PR itself valid?
+1.5 Are the testing criteria clear?
+(1.6 Does this PR respect the rights of previous authors?)
+(1.7 Is the PR a duplicate of another PR?)
+
+
+## 2. - Review the code (line-by-line)
```

Comment:
How many of these checks could be automated?

Reply:
Maybe we can put a note saying that the checks are in the process of/should progressively be automated? Using Cochinelle? And a similar note in section 4.4 about uncrustify?